web/client/themes/default/less/wizard.less

```diff
@@ -30,7 +30,7 @@
 
 
 .ms-wizard {
-    position: absolute;
+    // position: absolute;
```

Comment:
I don't think this is needed. Kindly check and revert this change

Reply:
@dsuren1 this was needed to solve a small bug, it came into limelight after Ellano's comment #2 on my previous PR [here](https://github.com/geosolutions-it/MapStore2/pull/9680#issuecomment-1798226903)
Interestingly this came into play as well for this backport. If we add up a long text, links etc. in editor for geostories via map, scrollbar doesn't appear. Kindly check it

Reply:
It occurs only if you add dashboard txt widget via map for a geostory. I was also confused with it and Stefano guided me how to reproduce it.

![image](https://github.com/geosolutions-it/MapStore2/assets/8802065/072411c2-5cbd-4751-9ab7-2e520bb80b8b)
![image](https://github.com/geosolutions-it/MapStore2/assets/8802065/e0bf2798-4804-4008-9b90-a087c9e0234c)